whipper/common/program.py

```diff
@@ -147,21 +129,15 @@ def getTable(self, runner, cddbdiscid, mbdiscid, device, offset,
 
     def getRipResult(self, cddbdiscid):
         """
-        Retrieve the persistable RipResult either from our cache (from a
-        previous, possibly aborted rip), or return a new one.
+        Return a RipResult object.
```

Comment:
Hm. I wonder if this means that we can no longer resume rips. :| (See also https://github.com/whipper-team/whipper/issues/136 )

Reply:
oops
`AttributeError: Program instance has no attribute 'saveRipResult'`

Reply:
Did another test rip, deleted it, started the rip again and control-c after a few tracks were ripped and starting it again didn't error out, so that's good.

Reply:
Hold on, doing another test rip+abort+restart and I'm saving the output this time.

Reply:
Here we go

https://gist.github.com/jtl999/d1d06b5730a858e912fdfccbe3aa461f